test/test-PreviewProvider.js

```diff
@@ -438,6 +442,7 @@ exports.test_copy_over_correct_data_from_firefox = function*(assert) {
 exports.test_compute_image_sizes = function*(assert) {
   let mockExperimentProvider = {data: {metadataService: false}};
   gPreviewProvider = new PreviewProvider(gMockTabTracker, gMockMetadataStore, mockExperimentProvider, {initFresh: true});
+  gPreviewProvider._store = {dispatch: () => {}};
```

Comment:
oooh. why not just reset it back at the end of `test_process_and_insert_links`? It should probably clean up for itself